src/app/app.component.ts

```diff
@@ -257,6 +257,11 @@ export class AppComponent implements OnInit {
         helpConfig.extends = this.config.runtime.params.get('svcid');
       }
 
+      if ((this.windowRef.nativeWindow as any).SKYUX_HOST && !helpConfig.locale) {
```

Comment:
I'd prefer to see `(this.windowRef.nativeWindow as any).SKYUX_HOST` be assigned to a `const` here to reduce the wordiness of the code.

```
const skyuxHost = (this.windowRef.nativeWindow as any).SKYUX_HOST;

if (skyuxHost && !helpConfig.locale) {
...
```